### PR TITLE
Dr. Borlaug's

### PR DESCRIPTION
**Pull request by azizunnabi**
In this mile stone 
1. Biography of Dr. Borlaug's
2. Used Html and Css

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,4 +70,6 @@ Give a ⭐️ if you like this project!
```
Comment (azizunnabi) [inline README.md:70]: Great job. I see that you have not added Description to your readme file. Please add the descritption. By this way your repo will look very professional. :smile:

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/test.md
```
Comment (azizunnabi) [inline .gitignore:1]: Please note that in the .gitignore file node_modules and test.md are wrongly structured. 
In this way you will not push your node_modules/ and test.md file.
It should be look like as following :+1: 
node_modules/
test.md

File: index.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <figure id="img-div">
+          <img
```
Comment (azizunnabi) [inline index.html:15]: Kindly do not use img in any ID or Class because this is not html/css good practice, :smile:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+            alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+          />
+          <figcaption id="img-caption">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico on
+            how to increase wheat yields - part of his life-long war on hunger.
+          </figcaption>
+        </figure>
+        <section id="tribute-info">
+          <h3 id="headline">Here's a time line of Dr. Borlaug's life:</h3>
```
Comment (azizunnabi) [inline index.html:26]: You should have to wrap all the ul tags in the article. By this way your code will look more professional. :+1:

File: style.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
+html {
+  /* Setting a base font size of 10px give us easier rem calculations
+     Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+   */
+  font-size: 10px;
```
Comment (azizunnabi) [inline style.css:5]: Please note that always try to remove the comments from your code, by this way your code will be look more professional and clean. :smile:

**Review by azizunnabi — COMMENTED** [review]
You have done the great job but I suggest you to check these changes and fix them

**Review by Kamba56 — APPROVED** [review]
Hi @azizunnabi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
JimCarreyYesSirGIF
## Highlights
- [x] Polite comments
- [ ] Great job using emojis
- [ ] Clear and understandable comments

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by azizunnabi** [comment]
Upon approval from the code reviewer it is here by merged with the main branch

**Pull request merged**